src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -135,11 +135,12 @@ public void setGame(final IGame game) {
     m_game = game;
     if (game != null) {
       game.addGameStepListener(m_gameStepListener);
-      gameStepChanged(game.getData().getSequence().getStep().getName(), game.getData().getSequence().getRound());
+      final String stepName = game.getData().getSequence().getStep().getName();
```

Comment:
Added unused local variable `stepName` to preserve any possible side effects caused by the call to `game.getData().getSequence().getStep().getName()`.